File: 2026-01/src/day_06/boj_5543/Main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int burger = 2000;
    int beverage = 2000;

    for (int i = 0; i < 3; i++) {
        int x;
        cin >> x;
        burger = min(burger, x);
    }

    for (int i = 0; i < 2; i++) {
        int x;
        cin >> x;
        beverage = min(beverage, x);
    }

    cout << burger + beverage - 50;
}

File: 2026-01/src/day_06/boj_5543/Main.java
package day_06.boj_5543;

import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int burger = 2000;
        int beverage = 2000;

        for (int i = 0; i < 3; i++) {
            burger = Math.min(burger, Integer.parseInt(br.readLine()));
        }

        for (int i = 0; i < 2; i++) {
            beverage = Math.min(beverage, Integer.parseInt(br.readLine()));
        }

        System.out.println(burger + beverage - 50);
    }
}
